install/kubernetes/Makefile

```diff
@@ -23,7 +23,6 @@ DEV_VERSION_REGEX := '[0-9]\+\.[0-9]\+-dev'
 CILIUM_CHART_REGEX := '\([vV]ersion:\) '$(VERSION_REGEX)
 CILIUM_PULLPOLICY_REGEX := '\([pP]ullPolicy:\) .*'
 QUICK_OPTIONS := \
-    --set hubble.tls.auto.enabled=false \
```

Comment:
Reviewer note: We used this value here to avoid storing certificate private keys in the pre-generated `quick-install.yaml`. With the other changes in this PR this is no longer necessary, thus I removed it.